knpu/travisci.md

```diff
@@ -46,6 +46,47 @@ file - `.travis.yml`. Paste!
 
 [[[ code('4a9e4ad713') ]]]
 
+***TIP
+You can use Github Actions as an alternative. There's an configuration example:
```

Comment:
```suggestion
You can use GitHub Actions as an alternative to Travis CI. Here's a configuration example:
```

Reply:
@bocharsky-bw I try this configuration here : https://github.com/abdounikarim/lorem-ipsum-bundle